src/require/createResolveRequest.js

```diff
@@ -0,0 +1,218 @@
+import { sep } from 'path';
+
+import resolve from 'resolve';
+import escapeStringRegexp from 'escape-string-regexp';
+
+import { initGetDependencies, initGetDependenciesFromPath } from './manageDependencies';
+import createPathRegExp from './createPathRegExp';
+
+export default function createResolveRequest(exports, directory, dependencyContext) {
+    const log = require('debug')('roc:core:require'); // eslint-disable-line
+
+    const contextCache = {};
+    const pattern = /^([^\/]*)\/?([^\/]*)/;
```

Comment:
Maybe better name for a variable would be information what this pattern is looking for?